charts/newrelic-k8s-metrics-adapter/templates/_helpers.tpl

```diff
@@ -99,12 +7,12 @@ Select a value for the region
 {{- if .Values.config.region -}}
   {{- .Values.config.region | upper -}}
 {{- else if .Values.global -}}
-  {{- if .Values.global.nrStaging -}}
+  {{- if (include "common.nrStaging" .) -}}
     Staging
-  {{- else if eq (include "newrelic-k8s-metrics-adapter.licenseKey" . | substr 0 2) "eu" -}}
+  {{- else if eq (include "common.license._licenseKey" . | substr 0 2) "eu" -}}
     EU
     {{- end }}
-{{- else if eq (include "newrelic-k8s-metrics-adapter.licenseKey" . | substr 0 2) "eu" -}}
+{{- else if eq (include "common.license._licenseKey" . | substr 0 2) "eu" -}}
```

Comment:
Regarding the not returning US is because if no region is passed, then will always be US.
Regarding the other 2 points, you are right and I corrected it.